client/my-sites/stats/controller.jsx

```diff
@@ -268,8 +265,9 @@ module.exports = {
 		} else if ( ! activeFilter || -1 === validModules.indexOf( context.params.module ) ) {
 			next();
 		} else {
-			if ( 'object' === typeof( site.options ) && 'undefined' !== typeof( site.options.gmt_offset ) ) {
-				momentSiteZone = i18n.moment().utcOffset( site.options.gmt_offset );
+			const gmt_offset = getSiteOption( context.store.getState(), siteId, 'gmt_offset' );
```

Comment:
This should be camel-case `gmtOffset`:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions

Only exception is in the case of REST API properties, like the ones stored in `state.sites.items`